splice_access_api/src/main/java/com/splicemachine/access/configuration/DDLConfiguration.java

```diff
@@ -51,12 +51,16 @@
     public static final String DDL_DRAINING_MAXIMUM_WAIT = "splice.ddl.drainingWait.maximum";
     private static final long DEFAULT_DDL_DRAINING_MAXIMUM_WAIT = 100000;
 
+    public static final String MERGE_REGION_WAIT_TIMEOUT = "merge.region.wait.timeout";
```

Comment:
Could we make it "splice.merge..."? Since this goes in the hbase-site.xml it could be confusing without the splice prefix.